fed/proxy/barriers.py

```diff
@@ -27,8 +28,14 @@
 import fed.utils as fed_utils
 from fed._private import constants
 from fed._private.grpc_options import get_grpc_options, set_max_message_length
+from fed._private.compatible_utils import _compare_version_strings
 from fed.config import get_cluster_config
-from fed.grpc import fed_pb2, fed_pb2_grpc
+if _compare_version_strings(importlib.metadata.version('protobuf'), '4.0.0'):
```

Comment:
This return False if ``version == 4.0.0``

Reply:
There is no version 4.0.0 of protobuf. The 4.0.0 version has been removed from pypi, so there is no need to make any modifications here.